modules/ui/TrainingTab.py

```diff
@@ -349,9 +349,24 @@ def __create_unet_frame(self, master, row):
         components.switch(frame, 5, 1, self.ui_state, "rescale_noise_scheduler_to_zero_terminal_snr")
 
         # max noising strength
-        components.label(frame, 6, 0, "Max Noising Strength",
+        components.label(frame, 6, 0, "Min Noising Strength",
+                         tooltip="Specifies the minimum noising strength used during training. This can help to improve composition, but prevents finer details from being trained")
+        components.entry(frame, 6, 1, self.ui_state, "min_noising_strength")
+
+        # max noising strength
+        components.label(frame, 7, 0, "Max Noising Strength",
                          tooltip="Specifies the maximum noising strength used during training. This can be useful to reduce overfitting, but also reduces the impact of training samples on the overall image composition")
-        components.entry(frame, 6, 1, self.ui_state, "max_noising_strength")
+        components.entry(frame, 7, 1, self.ui_state, "max_noising_strength")
+
+        # noising weight
+        components.label(frame, 8, 0, "Noising Weight",
+                         tooltip="Controls the emphasis on certain noise levels during training. A value of 0 disables this feature, leading to a uniform distribution where all noise levels are equally likely. Positive values increase the likelihood of selecting higher noise levels at any particular training step (resulting in smoother, low-frequency details), while negative values favor lower noise levels (sharper, high-frequency details). Note that extreme values (like -10 or 10) create a strong emphasis, significantly changing the focus of training.")
```

Comment:
Can you add the actual equation for the distribution function to the tooltip? It won't be very user friendly, but at least better than nothing.